frameworks/cassandra/tests/test_overlay.py

```diff
@@ -43,6 +45,7 @@ def teardown_module(module):
 @pytest.mark.sanity
 @pytest.mark.smoke
 @pytest.mark.overlay
+@pytest.mark.skipif('os.environ.get("SECURITY") == "strict"')
```

Comment:
I don't find the pytest documentation useless at all. Google "pytest skipif" will take you here, which has plenty of great examples:
https://docs.pytest.org/en/latest/skipping.html#id1